code/datums/components/gunpoint.dm

```diff
@@ -195,6 +210,28 @@
 		)
 		INVOKE_ASYNC(src, PROC_REF(trigger_reaction))
 
+///Shows if the parent is holding someone at gunpoint
+/datum/component/gunpoint/proc/examine(datum/source, mob/user, list/examine_list)
+	SIGNAL_HANDLER
+	examine_list += span_boldwarning("[parent] [parent.p_are()] holding [target] at gunpoint with [weapon]!")
```

Comment:
This line shouldn't be here if the viewer can't see the target.

Reply:
Is `can_see()` proc valid here? It doesn't seem to work correctly on corners, when the shooter is visible on screen, but it's not for the proc. What else is there to use?

Reply:
Yeah, just use `viewers()`. `can_see()` uses a step-towards method / line of sight and it's terrible as the doc itself says. I wonder why we aren't removing it yet.